[BHW-7]User authentication module

SugashiniKaliappan opened the pull request:
Sample Working screenshot
![Uploading Screen Shot 2025-02-27 at 9.54.07 PM.png…]()

SugashiniKaliappan committed:
user authentication modeule

SugashiniKaliappan committed:
user authentication modeule

FadhilAsh commented:
Good to go

SugashiniKaliappan committed:
latest changes

SugashiniKaliappan committed:
latest changes

tressapaulson reviewed (changes requested):
Looks good overall! Just a small suggestion
1. Add comments in each module
2. Add screenshot of executed entry
3. Standardized HTTP status codes: Use 403 Forbidden for missing tokens and 401 Unauthorized for invalid or expired tokens in authController.js

SugashiniKaliappan committed:
removing node modules

SugashiniKaliappan committed:
removing node modules

RishabSajith commented on user_authentication/routes/routes.js:
```diff
@@ -0,0 +1,48 @@
+*/
+export const ROUTES = [
+  {
+    url: "/api/v1/warddmissions",
```
Everything looks great!
Just came across a typo here for

SugashiniKaliappan replied:
updated

SugashiniKaliappan committed:
comments

RishabSajith reviewed (approved):
Thanks for addressing the typo!
Looks good to me

SugashiniKaliappan committed:
review comments

SugashiniKaliappan committed:
Merge branch 'main' into BHW_7/user_authentication

SugashiniKaliappan commented:
> Looks good overall! Just a small suggestion
> 
> 1. Add comments in each module
> 2. Add screenshot of executed entry
> 3. Standardized HTTP status codes: Use 403 Forbidden for missing tokens and 401 Unauthorized for invalid or expired tokens in authController.js

Addressed all comments

tressapaulson reviewed (approved):
Everything looks great!
